src/components/Selects/Select.tsx

```diff
@@ -120,49 +163,82 @@ export const Select: FC<SelectProps> = ({
+        >
+          <span className={clsx('text-black', { 'text-sm': isFilter })}>
+            {text}
+          </span>
         </span>
-        <span className="flex h-full items-center justify-center bg-black px-3">
-          <ChevronDownIcon
-            className={`ease h-5 w-5 transform fill-white stroke-white duration-300 ${
-              isOpen ? 'rotate-180' : ''
-            }`}
-          />
+        <div className="min-w-fit" />
+        <span
+          className={clsx('h-full basis-1/4 items-center justify-center px-3', {
+            'bg-lightBlue': isFilter,
+            'bg-black': !isFilter,
+          })}
+        >
+          <SelectChevron isFilter={isFilter} isOpen={isOpen} />
         </span>
       </button>
 
       {isOpen && (
-        <div className="absolute top-14 left-0 w-full rounded-md border-[0.5px] border-solid border-cobalt bg-white p-5 shadow-md">
+        <div
+          className={clsx(
+            'absolute left-0 transform rounded-md border-[0.5px] border-solid border-cobalt bg-white shadow-md',
+            {
+              'w-full translate-y-14 p-5': !isFilter,
+              'w-60 translate-y-2 p-3': isFilter,
+            },
+          )}
+        >
           {isSearchable && (
             <div className="relative mb-5">
               <input
                 onChange={(e) => setSearchQuery(e.currentTarget.value)}
                 type="text"
-                className="relative h-7 w-full border border-solid border-black px-3 py-1 pl-8 focus:outline-none focus:ring-2 focus:ring-blue-400"
-                placeholder="Search..."
+                className={clsx(
+                  'border-width-1 relative h-7 w-full rounded border border-solid border-black px-3 py-1 pl-8 focus:outline-none focus:ring-2 focus:ring-blue-400',
+                )}
+                placeholder="Search"
               />
-              <MagnifyingGlassIcon className="absolute top-1.5 left-2 h-4 w-4" />
+              <MagnifyingGlassIcon className="absolute top-1.5 left-2 h-4 w-4 fill-darkGray" />
             </div>
           )}
           <div
             className={`${
-              containerHeight ? `max-h-[${containerHeight}]` : 'max-h-fit'
-            } w-full overflow-x-visible overflow-y-scroll p-0`}
+              containerHeight ? `max-h-[240px]` : 'max-h-fit'
+            } w-full overflow-y-auto overflow-x-visible p-0`}
           >
             {isStringArray(options) ? (
-              <ul className="m-auto flex w-11/12 flex-col gap-4">
+              <ul
+                className={clsx('m-auto flex w-11/12 flex-col', {
+                  'gap-4': !isFilter,
+                  'gap-1': isFilter,
+                })}
+              >
                 {options.filter(handleSearchFilter).map((option) => (
                   <li key={option} className="flex items-center text-black">
-                    <span>{option}</span>
+                    <span className={clsx({ 'text-sm': isFilter })}>
```

Comment:
```suggestion
                    <span className={clsx({ 'text-md': isFilter })}>
```
Double check Figma. Also it seems like the font is `Inter`.